### PR TITLE
add ut of ads_stream.go

### PR DESCRIPTION
**Pull request by LiZhenCheng9527**
**What type of PR is this?**
/kind enhancement

**What this PR does / why we need it**:
add ut of ads_stream.go

### CODE REVIEW

File: pkg/controller/envoy/ads_stream_test.go [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+					func(ctx context.Context, opts ...grpc.CallOption) (service_discovery_v3.AggregatedDiscoveryService_StreamAggregatedResourcesClient, error) {
+						return nil, errors.New("fail to create adsstream")
+					})
+			},
+		},
+	}
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			tt.beforeFunc()
+			err := adsStream.AdsStreamCreateAndSend(client, context.Background())
```
Comment (hzxuzhonghu) [inline pkg/controller/envoy/ads_stream_test.go:56]: This is not right. There is no server at all. Make no sense to me.

**Comment by LiZhenCheng9527** [comment]
cancel /hold

**Comment by LiZhenCheng9527** [comment]
cc @nlgwcy

**Pull request merged**